presto-docs/src/main/sphinx/release/release-0.224.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.224
+=============
+
+General Changes
+---------------
+
+* Fix an issue where CPU time and scheduled time are swapped in query plans.
+* Fix a correctness issue for queries containing nested coercible type cast.
+* Add support for recoverable grouped execution when writing unpartitioned table.
```

Comment:
Is there some session variable for this feature?

Reply:
No there is not. As long as the table writer concurrency in the stage is 1, we are able to execute in recoverable mode.